### DISCUSSION TITLE
How do I import a repository with issues, milestones, etc from Gitlab to Github (node-gitlab-2-github doesn't work)

### DISCUSSION THREAD

**Discussion by DavidGriffith** [community/community#67437] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to figure out how I can import a repository from Gitlab to Github, preserving issues, milestones, pull-requests, etc.  Gitlab has a simple button to import into it, so why doesn't Github have something similar?  I've tried https://github.com/piceaTech/node-gitlab-2-github, but I keep running into API calls-per-hour limits and it had no means of backing off to be nice to the Github servers.

I've also seen https://github.com/actions/importer-issue-ops and https://github.com/actions/importer-labs/ as suggested solutions, but it's very unclear to me how to use those.

**Comment by DavidGriffith** [comment]
I've discovered https://github.com/gavinr/github-csv-tools/, but this doesn't work either.  It throws an error and aborts after importing the first issue.  See https://github.com/gavinr/github-csv-tools/issues/108 for my issue report on that.

**Reply by AARP41298** [reply]
i tried with github cli but the uses are limited
cypress too, but blocked due creating issues too fast

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! Check out [this discussion](https://github.com/cli/cli/discussions/9269) as it has another suggestion 😄
